src/tribol/geom/ContactPlane.cpp

```diff
@@ -1439,7 +1440,9 @@ bool ContactPlane3D::computeLocalInterpenOverlap()
 
          if (k > 2)
          {
-            SLIC_ERROR ("ContactPlane3D::computeInterpenOverlap : too many segment-plane intersections.");
+            SLIC_INFO("ContactPlane3D::computeInterpenOverlap(): too many segment-plane intersections.");
```

Comment:
Gut-check: Should this be a `SLIC_INFO`? Would `SLIC_DEBUG` work?

Now that it's no longer marked as an error and the return type is `DEGENERATE_OVERLAP`, I'm wondering if this might lead to too many log messages in some cases

Reply:
Also, could/should we add more context about the specific error? 
There's not too much to go on here if we do encounter this problem.

Reply:
I changed to `SLIC_DEBUG` which it should be, and elaborated as much as I can on the note. If this does get triggered it will be one of those gross things to debug.